app/models/folio/omniauth/authentication.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class Folio::Omniauth::Authentication < Folio::ApplicationRecord
+  self.table_name = "folio_omniauth_authentications"
```

Comment:
proc ne standardne pres module?

Reply:
Nefungovalo mi to.

Reply:
Upraveno skrz inicializer `folio/config/initializers/omniauth.rb`